runtime/mainnet/src/benchmarks.rs

```diff
@@ -42,3 +67,183 @@ frame_benchmarking::define_benchmarks!(
 	[pallet_nfts, Nfts]
 	[pallet_assets, TrustBackedAssets]
 );
+
+parameter_types! {
+	pub ExistentialDepositAsset: Option<Asset> = Some((
+		RelayLocation::get(),
+		ExistentialDeposit::get()
+	).into());
+	pub const AssetHubParaId: ParaId = ParaId::new(1000);
+}
+
+impl pallet_xcm::benchmarking::Config for Runtime {
+	type DeliveryHelper = (
+		polkadot_runtime_common::xcm_sender::ToParachainDeliveryHelper<
```

Comment:
should this also have a `ToParentDelivery`?

https://github.com/polkadot-fellows/runtimes/blob/main/system-parachains/asset-hubs/asset-hub-polkadot/src/lib.rs#L1128

Reply:
It will be better with it :+1:  [375e9d1](https://github.com/r0gue-io/pop-node/pull/479/commits/375e9d194d8a69a2e2ca9194c1594d189ea703e0)